packages/pages/src/dev/server/ssr/getLocalData.ts

```diff
@@ -18,10 +18,12 @@ export interface LocalDataManifest {
     {
       // The featureName for a specific static template
       featureName: string;
-      // The return value of the template's getPath()
-      staticURL: string;
-      // The locale codes
-      locales: string[];
+      staticPages: {
```

Comment:
static page `getPath` return val could use the locale.  So, we can't just have one staticURL per feature name anymore.